packages/amplify-graphql-api-construct-tests/src/__tests__/deploy-velocity/replace-2-gsis-update-attr-100k-records.test.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  API_POST_PROCESSOR_SET_PROVISIONED_THROUGHPUT,
+  COUNT_100_THOUSAND,
+  DURATION_30_MINUTES,
+  MUTATION_FOUR_FIELD_CREATE,
+  SCHEMA_FOUR_FIELDS_FINAL_TWO_INDEXED,
+  SCHEMA_FOUR_FIELDS_INITIAL_TWO_INDEXED,
+} from './deploy-velocity-constants';
+import { recordCountDataProvider, recordCountDataValidator, testManagedTableDeployment } from './deploy-velocity-test-core';
+
+testManagedTableDeployment({
+  name: 'Replace 2 GSIs updated w/ attr update - 100k Records',
```

Comment:
Looks like the only difference between `replace-2-gsis` and `replace-2-gsis-update-attr` is the provision throughput change. The intial and updated schema are the same for these two cases. I think these two can be merged to one suite

Reply:
Also I wonder the test result for this update. The initial table state is `PAY_BY_REQUEST` by default. Is it possible to change the index throughput without changing the table billing mode, which should be `PROVSIONED`?

Reply:
Yeah, looks like that's failing right now. Debugging locally, but I'll update here as well.

Reply:
I'm leaving these as two separate cases since I want to ensure we have independent signal for `gsi-replacement` and `gsi-replacement w/ additional attribute updates`